src/components/MovieTile/MovieTile.tsx

```diff
@@ -0,0 +1,55 @@
+import './MovieTile.css';
+import MovieTileModel from '../../models/MovieTileModel';
+import DotsIcon from './../../resources/DotsIcon.png';
+import { useState } from 'react';
+
+interface MovieTileProps {
+  movieModel: MovieTileModel;
+  onClickCallback: (movieName: string) => void;
+  onEditClickCallback: (movieName: string) => void;
+  onDeleteClickCallback: (movieName: string) => void;
+}
+
+function MovieTile({ movieModel, onClickCallback, onEditClickCallback, onDeleteClickCallback }: MovieTileProps) {
+    const [isContextMenuOpen, setContextMenuOpen] = useState(false);
+
+    const toggleContextMenu = () => {
+        setContextMenuOpen(!isContextMenuOpen);
+    };
+        
+    const onClick = () => {
+        onClickCallback && onClickCallback(movieModel?.MovieName);
+    };
+
+    const handleEditClick = () => {
+        onEditClickCallback && onEditClickCallback(movieModel?.MovieName);
+    };
+
+    const handleDeleteClick = () => {
+        onDeleteClickCallback && onDeleteClickCallback(movieModel?.MovieName);
+    };
+    return (
+        <div className="div-MovieTile-General">
+            <div onClick={onClick} className="div-MovieTile-Select">
+                <div className="div-MovieTile-imageContainer">
+                    <img src={movieModel?.ImageUrl} alt='Load error'/>
```

Comment:
The alt text provides information for an image if a user for some reason cannot view it (for example if the user uses a screen reader). So I guess better to change the text to "Movie poster".